.gitignore

```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
```

Comment:
please delete this file.We don't need any `.gitignore` file for this repository for now